README.md

```diff
@@ -178,14 +188,75 @@ applications:
   - conjur
 ```
 
+In PCF version 2.0+, when the service broker creates the Host identity for your application
+in Conjur, it will automatically add it to a Conjur Layer representing the `Organization`
+and `Space` the application is deployed into on PCF. This allows `Spaces` to be privileged
+and any apps deployed into that space to inherit those privileges.
+
+To accomplish this the service broker requires that the Org and Space policy exist prior
+to binding applications with Conjur. The policy to define an organizaiton and space is:
+```yaml
+---
+# Policy for the Organization
+- !policy
+  # Organization GUID from PCF.
+  # This may be obtained by running `cf org --guid {org name}
+  id: cbd7a05a-b304-42a9-8f66-6827ae6f78a1
+  body:
+    # Layer to allow privileging an entire organzation to a resource
+    - !layer
+
+    # Policy for the Spaceß
```

Comment:
Looks like a beta character snuck in on the end of this line somehow.

Reply:
🤦‍♂️ Thanks!

With I accidentally hit option-s instead of cmd-s to save, that's what get's inserted. I will remove them characters.